pkg/model/volumes.go

```diff
@@ -23,6 +23,9 @@ import (
 )
 
 func (dev *Dev) translateDeprecatedVolumeFields() error {
+	if dev.WorkDir == "" && dev.MountPath == "" && len(dev.Sync.Folders) == 0 {
+		dev.WorkDir = "/okteto"
```

Comment:
Will this also make /okteto the sync folder?

Reply:
Yeap